insights/specs/__init__.py

```diff
@@ -494,6 +494,7 @@ class Specs(SpecSet):
     ntptime = RegistryPoint()
     numa_cpus = RegistryPoint(multi_output=True, no_obfuscate=['hostname', 'ip'])
     numeric_user_group_name = RegistryPoint(no_obfuscate=['hostname', 'ip'])
+    nvidia_smi_l = RegistryPoint()
```

Comment:
My bad, as "NO IP/Hostname obfuscation" is selected for this spec, please add the `no_obfuscate=['hostname', 'ip']` here.

Reply:
Updated